<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>search all drives</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create_symlinks">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>create symlinks</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_search_result">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>541</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_src_dir">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>select dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_selected_src_dir">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search_dir">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>search dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_tgt_dir">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>select dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_selected_tgt_dir">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
